### PR TITLE
web:Keycloak auth

### PR DESCRIPTION
**Pull request by deetz99**
issue: #16 

- use keycloak instead of firebase auth
- cleanup all config and files using firebase auth
- move content file from yml to md

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-ar-dev--pr-38-aj4t0zut.web.app

File: web/site/devops/vaults.env [hunk 1]
```diff
@@ -7,4 +7,9 @@ AUTH_API_KEY="op://firebase/$APP_ENV/base/API_KEY"
...
 AUTH_PROJECT_ID="op://firebase/$APP_ENV/base/PROJECT_ID"
 AUTH_MESSAGING_SENDER_ID="op://firebase/$APP_ENV/business-ar/MESSAGING_SENDER_ID"
-AUTH_APP_ID="op://firebase/$APP_ENV/business-ar/APP_ID"
\ No newline at end of file
+AUTH_APP_ID="op://firebase/$APP_ENV/business-ar/APP_ID"
+
+#vaults keycloak
+NUXT_KEYCLOAK_AUTH_URL="op://keycloak/$APP_ENV/base/KEYCLOAK_AUTH_BASE_URL"
+NUXT_KEYCLOAK_REALM="op://keycloak/$APP_ENV/base/KEYCLOAK_REALMNAME"
+NUXT_KEYCLOAK_CLIENTID="op://keycloak/$APP_ENV/business-ar/UI_KEYCLOAK_RESOURCE_NAME"
```
Comment (deetz99) [inline web/site/devops/vaults.env:15]: Is this the correct setting? I have keycloak running with my local env but in the /gcbrun its failing with "missing client id"

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-ar-dev--pr-38-aj4t0zut.web.app

**Pull request merged**